backend/tasks/validation.py

```diff
@@ -172,10 +172,28 @@ def validate_option(options, answer):
     return False
 
 
-def validate_row_data(session, col, answer, question, adm):
+def validate_row_data(
+    session, col, answer, question, adm, valid_deps, answer_deps
+):
     default = {"error": ExcelError.value, "cell": col}
+    if (not answer != answer) and (
```

Comment:
We have too complex if, e.g: we can do `if answer == answer` instead `not`.
`0 == 0, True == True, "something" == "something"` all of them is true.
Only `np.nan == np.nan` is False, that's why you see `if answer != answer` @ifirmawan 
Please simplify this.

Reply:
noted, Let me try to fix it. Thank you mas